### PR TITLE
HomePage

### PR DESCRIPTION
**Pull request by anshtripathi01**
**Please Review**

### CODE REVIEW

**Review by shashikantmaurya0007 — APPROVED** [review]
i dont see any problem here

File: src/Components/Footer.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+<p>Made With Love By Ansh</p>
+<ul className="footer-social">
+    <li className="footer-link"><a href="https://www.linkedin.com/anshtripathi01" target="blank"><i
+                className="fab fa-linkedin fa-lg" aria-hidden="true"></i>
+        </a></li>
+    <li className="footer-link"><a href="https://www.github.com/anshtripathi01" target="blank"><i
+                className="fab fa-github fa-lg" aria-hidden="true"></i>
+        </a></li>
+    <li className="nav-link"><a href="https://www.twitter.com/ansh_tripathi01"><i className="fab fa-twitter fa-lg"
+                aria-hidden="true"></i> </a></li>
```
Comment (agrawalshanu708) [inline src/Components/Footer.js:17]: format the code

File: src/pages/Home/Home.module.css [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+}
+
+/* meadia Query for responsiveness */
+@media (max-width:972px) {
+    
+    .headerContainer {
+        flex-wrap: wrap;
+        margin: 4rem;
+        padding: 0 10.5rem;
+        background-color: #fff1f2;
```
Comment (agrawalshanu708) [inline src/pages/Home/Home.module.css:72]: use color variable

**Review by agrawalshanu708 — APPROVED** [review]
code structure is good
you can merge

File: src/Components/Footer.js (hunk 1 above)
Comment (anshtripathi01) [reply src/Components/Footer.js:17]: ok

**Pull request merged**